intro.Rmd

```diff
@@ -28,17 +28,31 @@ But packages are useful even if you never share your code. As Hilary Parker says
 
-Together, devtools and RStudio insulate you from the low-level details of how packages are built.  As you start to develop more packages, I highly recommend that you learn more about those details. The best resource for the official details of package development is always the official [writing R extensions][r-ext] manual. However, this manual can be hard to understand if you're not already familiar with the basics of packages. It's also exhaustive, covering every possible package component, rather than focussing on the most common and useful components, as this book does. Writing R extensions is a useful resource once you've mastered the basics and want to learn what's going on under the hood.
+Devtools works hand-in-hand with RStudio, which we believe is the best development environment for most R users. The main alternative is [Emacs Speaks Statistics](http://ess.r-project.org/) (ESS), which is a rewarding environment if you're willing to put in the time to learn Emacs and customise it to your needs. The history of ESS stretches back over 20 years (predating R!), but it's still actively developed and many of the workflows described in this book are also available there.
+
+:::rstudio-tip
+Through the book, we highlight specific ways that RStudio can expedite your package development workflow, in specially formatted sections like this.
+:::
+
+Together, devtools and RStudio insulate you from the low-level details of how packages are built. As you start to develop more packages, we highly recommend that you learn more about those details. The best resource for the official details of package development is always the official [writing R extensions][r-ext] manual. However, this manual can be hard to understand if you're not already familiar with the basics of packages. It's also exhaustive, covering every possible package component, rather than focussing on the most common and useful components, as this book does. Writing R extensions is a useful resource once you've mastered the basics and want to learn what's going on under the hood.
+
+## Something something something
+
+Version 2.0.0 of devtools is the first major release since Jim Hester assumed the role of devtools' maintainer in February 2018.
+
+TODO: describe the conscious uncoupling (https://www.tidyverse.org/articles/2018/10/devtools-2-0-0/).
```

Comment:
This could also go in the preface where we detail the changes from the previous edition of the book.